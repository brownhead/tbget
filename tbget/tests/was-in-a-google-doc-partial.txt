{'double encoding is great': 'super', 'traceback': {'traceback': 'Traceback (most recent call last):\n  File “tools/devshell.py”, line 145, in <module>\n    from devshell_eval import *\n  File
 “/Users/johnsullivan/khan/webapp/tools/devshell_eval.py”, line 4, in <module>\n    from assessment_items.models import *\n  File “/Users/johnsullivan/khan/webapp/assessment_items/models.py”, line 16, in <module>\n    import api.jsonify\n  File "/Users/johnsullivan/khan/webapp/api/jsonify.py", line 21, in <module>\n    import frozen_model\n  File "/Users/johnsullivan/khan/webapp/frozen_model.py", line 16, in <module>\n    import content.models\n  File “/Users/johnsullivan/khan/webapp/content/models.py”, line 77, in <module>\n    from intl import i18n\n  File "/Users/johnsullivan/khan/webapp/intl/i18n.py", line 43, in <module>\n    import intl.models\n  File "/Users/johnsullivan/khan/webapp/intl/models.py", line 16, in <module>\n    import intl.crowdin\n  File "/Users/johnsullivan/khan/webapp/intl/crowdin.py", line 15, in <module>\n    import intl.translate\n  File ‘/Users/johnsullivan/khan/webapp/intl/translate.py’, line 57, in <module>\n    import event_log\n  File "/Users/johnsullivan/khan/webapp/event_log.py", line 107, in <module>\n    from logging_tools import ua_logic\n  File "/Users/johnsullivan/khan/webapp/logging_tools/ua_logic.py", line 6, in <module>\n    from third_party import user_agents\n  File "/Users/johnsullivan/khan/webapp/third_party/__init__.py", line 137, in load_module\n    __import__(short_module_name)\n  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/__init__.py", line 3, in <module>\n    from .parsers import parse\n  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/parsers.py", line 4, in <module>\n    from ua_parser import user_agent_parser\n  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 484, in <module>\n    _v2_replacement))\n  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_pars
