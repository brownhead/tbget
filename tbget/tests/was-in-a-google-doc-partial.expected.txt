Traceback (most recent call last):
  File "tools/devshell.py", line 145, in <module>
    from devshell_eval import *
  File "/Users/johnsullivan/khan/webapp/tools/devshell_eval.py", line 4, in <module>
    from assessment_items.models import *
  File "/Users/johnsullivan/khan/webapp/assessment_items/models.py", line 16, in <module>
    import api.jsonify
  File "/Users/johnsullivan/khan/webapp/api/jsonify.py", line 21, in <module>
    import frozen_model
  File "/Users/johnsullivan/khan/webapp/frozen_model.py", line 16, in <module>
    import content.models
  File "/Users/johnsullivan/khan/webapp/content/models.py", line 77, in <module>
    from intl import i18n
  File "/Users/johnsullivan/khan/webapp/intl/i18n.py", line 43, in <module>
    import intl.models
  File "/Users/johnsullivan/khan/webapp/intl/models.py", line 16, in <module>
    import intl.crowdin
  File "/Users/johnsullivan/khan/webapp/intl/crowdin.py", line 15, in <module>
    import intl.translate
  File "/Users/johnsullivan/khan/webapp/intl/translate.pyâ€™", line 57, in <module>
    import event_log
  File "/Users/johnsullivan/khan/webapp/event_log.py", line 107, in <module>
    from logging_tools import ua_logic
  File "/Users/johnsullivan/khan/webapp/logging_tools/ua_logic.py", line 6, in <module>
    from third_party import user_agents
  File "/Users/johnsullivan/khan/webapp/third_party/__init__.py", line 137, in load_module
    __import__(short_module_name)
  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/__init__.py", line 3, in <module>
    from .parsers import parse
  File "/Users/johnsullivan/khan/webapp/third_party/user_agents/parsers.py", line 4, in <module>
    from ua_parser import user_agent_parser
  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_parser.py", line 484, in <module>
\n    _v2_replacement))\n  File "/Users/johnsullivan/khan/webapp/third_party/ua_parser/user_agent_pa ...
